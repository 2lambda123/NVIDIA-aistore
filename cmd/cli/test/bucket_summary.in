ais show bucket tmp // FAIL
ais create bucket $BUCKET_1
ais create bucket $BUCKET_2
ais show bucket $BUCKET_1
ais show bucket aiss://$BUCKET_1 // FAIL "invalid backend provider "aiss""

ais gen-shards "ais://$BUCKET_1/tmp/shard-{0..149}.tar" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais gen-shards "ais://$BUCKET_2/tmp/shard-{0..19}.tar" --fcount 10 --fsize 1KB --conc 1 --cleanup // IGNORE
ais show bucket $BUCKET_1
ais show bucket $BUCKET_2
