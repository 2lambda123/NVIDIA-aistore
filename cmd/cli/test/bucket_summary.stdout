"ais://$BUCKET_1" created
"ais://$BUCKET_2" created
NAME	 OBJECTS	 SIZE	 USED %
ais://$BUCKET_1	 0	 0B	 0.00%
NAME	 OBJECTS	 SIZE	 USED %
ais://$BUCKET_1	 150  375.00KiB	 0.00%
NAME	 OBJECTS	 SIZE	 USED %
ais://$BUCKET_2	 20  320.00KiB	 0.00%
